projects/u-mill/white_mesa/decantation_filtration.py

```diff
@@ -46,9 +46,9 @@
 
      2) Counter-Current Decantation (CCD)
        - # of thickeners:                    7
-       - volume per thickener:               339.29 m^3  (12 m diameter, 3 m height)
```

Comment:
Why are you removing this?

Reply:
This must have reverted back by accident. I had to redownload the entire repository due to the extent of conflicts in my version of the code. This is still in my current version of the code as far as I can tell.